sigmf-spec.md

```diff
@@ -91,7 +91,9 @@ Augmented Backus-Naur form (ABNF) is used as defined by [RFC
 
 Fields defined as "human-readable", a "string", or simply as "text" shall be
 treated as plaintext where whitespace is significant, unless otherwise
-specified.
+specified. Fields defined "human/machine-readable" should be a short, simple
+text string without whitespace" that can be readily understood by a human and
```

Comment:
I think we need to retain the comment that whitespace is significant, as it communicates that the expectation is that readers do not strip whitespace from `description` fields. What did you have in mind in pulling it out?

Also, you've got a stray `"` on this line 😅

Reply:
I'm not following this; I didn't remove lines 92/93 for `"human-readable", a "string", or simply as "text"` just added the statement for `"human/machine-readable"` and I do feel that we should keep this to without whitespace if it is indeed human and machine readable (can use `_` or other non-whitespace options).  Thoughts?

Definitely have an extra `"` though :).

Reply:
You're right. I just misread the diff because lol 🤦‍♂️

Spec is good as-is, pending typo change 😄